src/geom/cell_inf_hex8.C

```diff
@@ -67,7 +67,7 @@ const unsigned int InfHex8::edge_nodes_map[InfHex8::num_edges][InfHex8::nodes_pe
 const unsigned int InfHex8::edge_sides_map[InfHex8::num_edges][2] =
   {
     {0, 1}, // Edge 0
-    {1, 2}, // Edge 1
+    {0, 2}, // Edge 1
```

Comment:
Come again?  I could imagine having screwed up this table somehow (or having copied a mistake in the diagram in cell_inf_hex8.h?), but not like this.  An InfHex vertex has three edges connected to it.  There's no way node 0 is part of 4 edges.

Reply:
Sound logic. This is actually a map that I implemented :-) 

I'll look into it

Reply:
Oh - this is edge -> side map. Has nothing to do with nodes. It's valid.

Reply:
Actually, hold up. Need to correct it in the other classes, too.